docs/source/developer_guide/development_environment.md

```diff
@@ -12,5 +12,50 @@ dependencies and RAPIDS libraries automatically fetched and correctly versioned.
 script commands for simple building and testing of all RAPIDS libraries, including cuSpatial.
 `rapids-compose` is the recommended way to set up your environment to develop for cuspatial.
 
-For developers who would like to build from conda or from source, see
-[README.md](https://github.com/rapidsai/cuspatial/blob/main/README.md).
+## To build and install cuSpatial from source:
+
+### Pre-requisite
+
+- gcc >= 7.5
+- cmake >= 3.23
+- miniconda
+
+### Fetch cuSpatial repository
+
+```shell
+export `CUSPATIAL_HOME=$(pwd)/cuspatial` && \
+git clone https://github.com/rapidsai/cuspatial.git $CUSPATIAL_HOME
+```
+### Install dependencies
+
+1. `export CUSPATIAL_HOME=$(pwd)/cuspatial`
+2. clone the cuSpatial repo
+
+```shell
+conda env update --file conda/environments/all_cuda-115_arch-x86_64.yaml
```

Comment:
```suggestion
conda env update --file conda/environments/all_cuda-118_arch-x86_64.yaml
```

Reply:
Included this update on the new build.md file!